### PR TITLE
Add token for serving private orignal images

### PR DESCRIPTION
**Pull request by gava**

### CODE REVIEW

File: lib/Image/Auth/Token.php [hunk 1]
```diff
@@ -12,79 +12,118 @@
  * Class AuthToken
  * @package Wix\Mediaplatform\Image
  */
-class Token extends Option {
```
Comment (lelikg) [inline lib/Image/Auth/Token.php]: Something weird happened here

File: lib/Image/Auth/Token.php [hunk 2]
```diff
@@ -12,79 +12,118 @@
...
+     * @param $filePath string
+     * @param $imageHeight int
+     * @param $imageWidth int
+     *
+     * @return string
+     */
+    public static function createOriginalImageToken($appId, $appSecret, $filePath)
+    {
+
+        $token = new \Wix\Mediaplatform\Authentication\Token();
```
Comment (lelikg) [inline lib/Image/Auth/Token.php]: Lots of code duplication, please use extract method :)

Comment (gava) [reply lib/Image/Auth/Token.php]: Please show what do you mean. I will approve :-)

File: lib/Image/Auth/Token.php (hunk 1 above)
Comment (gava) [reply lib/Image/Auth/Token.php]: Code reformat with the IDE

**Pull request merged**